src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,127 +1,114 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
-    /**
-     * Верните текущую дату в виде строки в зависимости от запроса.
-     *
-     * @param datePart Запрос на часть даты или всю дата целиком:
-     *                 - FULL - текущая дата целиком год, месяц, день (число месяца)
-     *                 в формате YYYY-MM-DD, возвращаемое значение по умолчанию;
-     *                 - YEAR - текущий год;
-     *                 - MONTH - название текущего месяца;
-     *                 - DAY - текущий день (число месяца);
-     *                 В любом другом случае бросить DateTimeException
-     **/
+    private static final int FIRST_INDEX = 0;
+    private static final int SECOND_INDEX = 1;
+    private static final int THIRD_INDEX = 2;
+    private static final int MAX_MONTH = 12;
+    private static final int MAX_DAYS = 31;
+    private static final ZoneId ZONE = ZoneId.of("+02:00");
+    private static final LocalDate NOW = LocalDate.now();
+    private static final DateTimeFormatter NATIVE
+            = DateTimeFormatter.ofPattern("d MMM yyyy", Locale.ENGLISH);
+    private static final DateTimeFormatter FORMAT_TIME
+            = DateTimeFormatter.ofPattern("dd LLLL yyyy HH:mm", Locale.ENGLISH);
```

Comment:
```suggestion
    private static final DateTimeFormatter DATE_FORMATTER
            = DateTimeFormatter.ofPattern("d MMM yyyy", Locale.ENGLISH);
    private static final DateTimeFormatter DATE_TIME_FORMATTER
            = DateTimeFormatter.ofPattern("dd LLLL yyyy HH:mm", Locale.ENGLISH);
```

the names must be more informative